app/components/Onboarding/Syncing.js

```diff
@@ -9,10 +9,9 @@ import styles from './Syncing.scss'
 
 class Syncing extends Component {
   componentWillMount() {
-    const { fetchBlockHeight, newAddress } = this.props
+    const { fetchBlockHeight } = this.props
 
     fetchBlockHeight()
-    newAddress('np2wkh')
```

Comment:
For brand new nodes this PR doesn't fetch an address and I get a forever spinner while syncing. This is because `fetchInfo` (which then calls `walletAddress`) isn't called until the node is synced. The PR relies on there already being an address created. 

We should call `fetchInfo` here instead so that we go grab the `pubKey` and generate an address.

Reply:
Ah that actually won't work as it can get called before `gRPC` is connected. Let me think some more